corehq/apps/commtrack/views.py

```diff
@@ -270,29 +273,36 @@ class StockLevelsView(BaseCommTrackManageView):
     template_name = 'commtrack/manage/stock_levels.html'
 
     def get_existing_stock_levels(self):
-        """
-        Returns a list of dicts of this form:
-        {
-            'loc_type': 'district,
-            'emergency_level': 0.5,
-            'understock_threshold': 1.5,
-            'overstock_threshold': 3,
-        }
-        """
+        loc_types = LocationType.objects.by_domain(self.domain)
         return [{
             'loc_type': loc_type.name,
-            'emergency_level': 0.5,
-            'understock_threshold': 1.5,
-            'overstock_threshold': 3,
-        } for loc_type in self.domain_object.location_types]
+            'emergency_level': loc_type.emergency_level,
+            'understock_threshold': loc_type.understock_threshold,
+            'overstock_threshold': loc_type.overstock_threshold,
+        } for loc_type in loc_types]
```

Comment:
(not actionable unless you want) i like using `namedtuples` for stuff like this.